### ISSUE TITLE
Download is not working properly after updating to 4.0.2 

### ISSUE THREAD

**Issue by mediaessenz** [beechit/fal_securedownload#196]
The problem is the change came with commit 54b1a4513360d1111d91c451d622e654304daba3 in the FileDumpHook::fileDumpContens method:

```php
[pre commit]  $downloadName = $file->getProperty('download_name') ?: $file->getName();
[new] $downloadName = $file->hasProperty('download_name') ? $file->getProperty('download_name') : $file->getName();
```

The problem is, that $file->hasProperty('download_name') is alway true, if the sysext:filemetadata is installed.

Changing this line to the following solves the issue:
```php
$downloadName = $file->hasProperty('download_name') && $file->getProperty('download_name') ? $file->getProperty('download_name') : $file->getName();
```

**Comment by FamousWolf** [comment]
@mediaessenz Do you want to create a PR for this (so you get the credit) or do you want me to do it?

**Comment by mediaessenz** [comment]
I will do it :-)

**Comment by FamousWolf** [comment]
Thanks. I'll test and merge it tomorrow.

**Comment by FamousWolf** [comment]
Unfortunately I didn't have time for this yesterday. I've moved it to Friday.

**Comment by aWuttig** [comment]
@FamousWolf  Could you please release a new version 4.0.3 with the fix.

**Comment by FamousWolf** [comment]
@aWuttig I was waiting for a reply to a different issue, but they're not replying. So I just released 4.0.3.
